client/components/jetpack/card/jetpack-product-card/display-price/time-frame.tsx

```diff
@@ -17,11 +18,25 @@ const TimeFrame: React.FC< TimeFrameProps > = ( { expiryDate, billingTerm } ) =>
 		moment.isMoment( expiryDate ) && expiryDate.isValid() ? expiryDate : null;
 
 	const billingTermText = useMemo( () => {
+		const isNewFormat = isEnabled( 'jetpack/pricing-page-rework-v1' ); // TO-DO remove this flag once we have the new translation in production
```

Comment:
This can be something like `useCompactFormat` instead of something generic `isNewFormat`.

Reply:
Looks like we don't need this if we try to display the translations in separate block..